components/Match/renderMatch.js

```diff
@@ -0,0 +1,324 @@
+/* eslint-disable */
+import { isRadiant } from '../../utility';
+import constants from 'dotaconstants';
+import { generatePlayerAnalysis } from './analysis';
+
+function renderMatch(m)
+{
```

Comment:
part of the refactoring process should be upgrading to 'self-documenting' code with more descriptive variable names.